mappings/net/minecraft/client/gui/widget/ButtonWidget.mapping

```diff
@@ -10,3 +10,4 @@ CLASS net/minecraft/class_4185 net/minecraft/client/gui/widget/ButtonWidget
 	CLASS class_4241 PressAction
 		METHOD onPress (Lnet/minecraft/class_4185;)V
 			ARG 1 button
+	CLASS class_5316 TooltipAction
```

Comment:
not really an action, maybe `TooltipSupplier`

Reply:
Yeah, the reason I went for this is the method name is `onTooltip`. I think that will make more sense tho